sql-grammar.md

```diff
@@ -17,4 +17,6 @@ a[name]:focus {
 }
 </style>
 
+{{site.data.alerts.callout_success}}This page describes the full CockroachDB SQL grammar. However, as a starting point, it's best to reference our <a href="sql-statements.html">statement-specific pages</a> first, which provide detailed explanations and examples.{{site.data.alerts.end}}
```

Comment:
As our alerts are currently set up, you have to use html formatting inside them. There's a way to enable markdown inside html, though (an extra attribute). Let me see if that works.